source/draw/gpu/opengl/OpenGLWindow.ooc

```diff
@@ -13,7 +13,14 @@ use opengl
 use base
 
 version(!gpuOff) {
-OpenGLWindow: class extends OpenGLSurface {
+OpenGLWindow: class extends GpuCanvas {
+	_toLocal := FloatTransform3D createScaling(1.0f, -1.0f, -1.0f)
+	context ::= this _context as OpenGLContext
+	drawLines: override func ~explicit (pointList: VectorList<FloatPoint2D>, pen: Pen) { raise("Cannot draw lines directly to a window.") }
+	drawPoints: override func ~explicit (pointList: VectorList<FloatPoint2D>, pen: Pen) { raise("Cannot draw lines directly to a window.") }
+	flipMatrix: func -> FloatTransform3D {
+		FloatTransform3D createScaling(this _coordinateTransform a as Float, -(this _coordinateTransform e as Float), 0.0f)
+	}
```

Comment:
This is all that the previous transforms did since the viewport controlled the location.

@emilwestergren Do we even need this coordinate transform for displaying the final image?